### DISCUSSION TITLE
A Japanese API reference / APIリファレンスの日本語版について

### DISCUSSION THREAD

**Discussion by Derorin719** [growilabs/growi#8098] [Ideas / Requests] [answered]
Why isn't there a Japanese edition of the API reference?
APIリファレンスの日本語版はないのですか？

**Comment by yuki-takei** [comment, marked as answer]
The API reference will be maintained in English only as it is automatically generated from the source code.
APIリファレンスはソースコードから自動生成されるため、英語のみでの提供となります。
